PART5.txt

```diff
@@ -1226,23 +1226,15 @@ shall be a Supreme Court of India consisting of a Chief Justice of India and,
 until Parliament by law prescribes a larger number, of not more than seven 1
 other Judges.
 (2) Every Judge of the Supreme Court shall be appointed by the
-President by warrant under his hand and seal after consultation with such of the
-Judges of the Supreme Court and of the High Courts in the States as the
-President may deem necessary for the purpose and shall hold office until he
+President by warrant under his hand and seal on the recommendation of the National
+Judicial Appointments Commission referred to in article 124A and shall hold office until he
 attains the age of sixty-five years:
 
 ______________________________________________
 1. Now “thirty”, vide Act 11 of 2009, s. 2.
 
-52
 
-THE CONSTITUTION OF INDIA
-
-Provided that in the case of appointment of a Judge other than the Chief
-Justice, the Chief Justice of India shall always be consulted:
-Provided further that—
-(a) a Judge may, by writing under his hand addressed to the
```

Comment:
Does the resignation clause go away with the 99th?

Reply:
No, only the first provisio requiring Chief Justice Consultation goes away. The Second provisio re Judge resignation - the only change there is that the "provided further that" becomes "provided"